src/sentry/runner/commands/run.py

```diff
@@ -552,6 +552,7 @@ def ingest_consumer(consumer_types, all_consumer_types, **options):
 @click.option("--input-block-size", type=int, default=DEFAULT_BLOCK_SIZE)
 @click.option("--output-block-size", type=int, default=DEFAULT_BLOCK_SIZE)
 @click.option("--factory-name", default="default")
+@click.option("--output-topic", type=str, default=None)
```

Comment:
I am not sure managing as a CLI is the right option.
The first issue that comes to mind is that we configure topics in the server.py file (that is then overridden in all environments)
https://github.com/getsentry/sentry/blob/master/src/sentry/conf/server.py#L2322-L2381

Specifically having topic names in the CLI violates that abstraction over topic configs.  Fixing it would be a breaking changes for open source users.

More in general, this consumer will be deployed in multiple instances, for different use cases while maintaining the same logic (a bit like the snuba consumer). It seems there is a number of parameters that have to be configured for each:
- input topic
- output topic
- a tag or prefix for all the metrics it records
- the table to use 
- the use case id to inject in snuba messages
- probably more. 

I would suggest to have a proper data model for this config (as we do for storage keys in snuba) that can contain all the config parameters for an indexer in a way that can be overridden. Having all the parameters part of the CLI would be very error prone when setting the indexer up and it would also bloat the CLI a lot.

One option is to create a config in server.py (that can be overridden) and that is structured, then have a logical use case id that you would pass to the CLI to pick up all the parameters (like `release_health` vs `performance` vs etc.)

Another would be to create classes that represent the indexer config if that is useful in any way. You would still have to allow overrides through server.py.

Both require having at least a vague idea of the parameters we want to allow to select at startup.

Reply:
yeah, I'll update this once we have the performance metrics DB and we can fully inject the config changes